index.js

```diff
@@ -1,17 +1,57 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.sortItems();
+    this.currencyLength();
+  }
 
-  get(pos) {}
+  sortItems() {
+    this.items.sort((a,b)=>a - b);
+  }
 
-  max() {}
+  currencyLength() {
+    this.length = this.items.length;
+  }
 
-  min() {}
+  get(pos) {
+    if(this.items[pos]) {
+      return pos;
```

Comment:
Aqui vc pode retornar um ternário:

return this.items[pos] ? this.items[pos] : throw new Error('OutOfBounds');

Fica legal assim :-)